hazelcast/src/main/java/com/hazelcast/quorum/HeartbeatAware.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
```

Comment:
Shouldn't this be part of the heartbeat package?

Reply:
My initial thought was to define this interface `com.hazelcast.cluster` package, however it is only useful in `QuorumFunction`s (at least that's the current situation) so I opted for the `com.hazelcast.quorum` package... I am open to moving it to `com.hazelcast.cluster` if other reviewer(s) find it makes sense to have this interface there.